packages/flutter/lib/src/gestures/events.dart

```diff
@@ -1731,6 +1731,22 @@ abstract class PointerSignalEvent extends PointerEvent {
   });
 }
 
+/// A function used to send information about a PointerEvent back to the engine.
+///
+/// This is used on the web to `preventDefault`.
+typedef RespondPointerEventFn = void Function({required bool allowPlatformDefault});
```

Comment:
Can we avoid the Fn abbreviation here [1] and instead follow the naming rules [2]?

[1] https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-abbreviations
[2] https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#naming-rules-for-typedefs-and-function-variables

Reply:
Yes! in fact, I'll actually rename it to something similar to the  `ui.OnPointerDataRespondCallback` typedef.